nntrainer/dataset/databuffer_file.cpp

```diff
@@ -293,24 +293,6 @@ int DataBufferFromDataFile::setDataFile(DataType type, std::string path) {
     }
     test_name = path;
   } break;
-  case DATA_LABEL: {
-    std::string data;
-    validation[type] = true;
-    if (!data_file.good()) {
-      ml_loge("Error: Cannot open label file");
-      SET_VALIDATION(false);
-      return ML_ERROR_INVALID_PARAMETER;
-    }
-    while (data_file >> data) {
-      labels.push_back(data);
-    }
-    if (class_num != 0 && class_num != labels.size()) {
-      ml_loge("Error: number of label should be same with number class number");
-      SET_VALIDATION(false);
-      return ML_ERROR_INVALID_PARAMETER;
-    }
-    class_num = labels.size();
```

Comment:
so in case of file based databuffer, how is the number of class information provided?

Reply:
It is unified to get from the width of the output shape. Later this is being change to directly take label shape so that, number of classes is no longer needed(around #1395)

Reply:
how about instead of taking the width of the output shape, we consume the exact size of the output shape, especially for cases when output is not for classification purposes?

Reply:
Yes, similar thing is planned (we take label shape, if not given, taking output shape). For the knn we are planning to do, it takes single integer as a label and output is a vector.

Reply:
So, we don't care about the label data anymore. It makes sense.

Reply:
cool.